sources/packages/web/src/services/http/dto/ConfirmationOfEnrolment.dto.ts

```diff
@@ -53,6 +54,8 @@ export interface ApplicationDetailsForCOEAPIOutDTO {
   coeStatusClass: StatusChipTypes;
   maxTuitionRemittanceAllowed: number;
   hasOverawardBalance: boolean;
+  disabilityProfileStatus: DisabilityStatus;
+  disabilityApplicationStatus: DisabilityStatus;
```

Comment:
Are both the types in line 57 and 58 supposed to be the same?

Reply:
Good catch. Both should be strings.